pkgs/by-name/st/stats/package.nix

```diff
@@ -1,29 +1,44 @@
 {
   lib,
-  stdenvNoCC,
-  fetchurl,
-  undmg,
+  stdenv,
+  fetchFromGitHub,
   nix-update-script,
+  apple-sdk,
+  swift,
+  xcbuildHook,
 }:
 
-stdenvNoCC.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: {
   pname = "stats";
   version = "2.11.18";
 
-  src = fetchurl {
-    url = "https://github.com/exelban/stats/releases/download/v${finalAttrs.version}/Stats.dmg";
-    hash = "sha256-xEU897PE8LniQJijp83oSbHllck3pyediMqz/bzgFyE=";
+  src = fetchFromGitHub {
+    owner = "exelban";
+    repo = "Stats";
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-WjDg4urTLtQczM4qEco8IoVaYaXAKAS7gOuMsCrkRhc=";
   };
 
-  sourceRoot = ".";
+  nativeBuildInputs = [
+    apple-sdk
```

Comment:
This is redundant; `apple-sdk` is part of the standard environment (and Swift propagates its required SDK anyway).

Reply:
SDKs also go in `buildInputs` not `nativeBuildInputs` (unless you’re using an SDK for a `depsBuildBuild` compiler, which isn’t the case here).